.github/workflows/deploy_and_build_prod.yml

```diff
@@ -152,95 +237,3 @@ jobs:
           token: ${{ secrets.GH_TOKEN_SUPPLYALLY_BOT }}
           name: Release ${{ env.GITHUB_REF_SLUG }}
           allowUpdates: true
-  build_android:
-    needs: [deploy_prod, create_release]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: rlespinasse/github-slug-action@v3.x
-      - uses: expo/expo-github-action@v5
-        with:
-          expo-packager: npm
-          expo-username: ${{ secrets.EXPO_CLI_PROD_USERNAME }}
-          expo-password: ${{ secrets.EXPO_CLI_PROD_PASSWORD }}
-          expo-cache: true
-      - name: Install Packages
-        run: |
-          npm config set //npm.pkg.github.com/:_authToken $NPM_TOKEN
-          npm ci
-        env:
-          NPM_TOKEN: ${{ secrets.GH_TOKEN_SUPPLYALLY_BOT }}
-      - name: Build Android Release
-        env:
-          NODE_OPTIONS: --max_old_space_size=7168
-          SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
-          SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
-          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
-          APP_BUILD_VERSION: ${{ github.run_number }}
-          APP_BINARY_VERSION: ${{ needs.deploy_prod.outputs.latestBinaryVersion }}
-          DOMAIN_FORMAT: ${{ secrets.DOMAIN_FORMAT }}
-        run: |
-          expo build:android --release-channel=${{ needs.deploy_prod.outputs.releaseChannel }} > buildLogAndroid.txt
-          cat buildLogAndroid.txt
-      - name: Parse Asset URL
-        id: androidUrl
-        run: |
-          ASSET_URL=$(cat buildLogAndroid.txt | tail | egrep -o 'https?://expo\.dev/artifacts/[^ ]+')
-          echo The android url is $ASSET_URL
-          echo "::set-output name=assetUrl::$ASSET_URL"
-      - name: Download APK Asset
-        run: wget -O supplyally-${{ env.GITHUB_REF_SLUG }}.apk ${{ steps.androidUrl.outputs.assetUrl }}
-      - name: Upload Release Asset
-        uses: svenstaro/upload-release-action@v2
```

Comment:
so we no longer upload the assets on our repo ah?

Reply:
oops i guess we should have it

Reply:
hmm how important is it to do the release asset thing for prod ah? the asset url for eas looks like https://expo.dev/artifacts/eas/5PGQXo45xk8z3NG7uTDvSk.ipa but the problem is idk where “5PGQXo45xk8z3NG7uTDvSk” comes from. its not the build id or the commit id. its not anywhere in the build logs also. just looks like a random string. so idk how we can get that string dynamically in ci